src/styles/home.scss

```diff
@@ -281,6 +281,76 @@
   @include primary-cta;
 }
 
+// ---------------------------------------------------------------------------
+// Launch Sequences
+// ---------------------------------------------------------------------------
+
+
+  .launch__content {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 80px 0 40px 0;
+    width: 100%;
+  }
+  
+  
+  .launch__header {
+    @include h4-text;
+    color: $pale-yellow !important;
+  }
+  
+  .launch__countdown{
+    @media (min-width: $sm) {
+      font-size: 72px;
+    }
+    font-size: 40px;
```

Comment:
Check w Judy to see if there's a text include you can use instead of special casing it

Reply:
checked with judy
the font sizes for the countdown clock are specifically 60px, which is different from any in-built text sizes like h1